server/utils/formatting.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../config');
+
+const formatResponseBodyBySongInfo = songInfo => {
+    return {...songInfo,
+        uploaded_at: new Date(songInfo.published).toLocaleDateString(),
+        formats: getFormatContainersByFormats(songInfo.formats),
+        duration: convertSecondsToTime(songInfo.length_seconds),
+        channel_url: `${config.youtube.channel_url}/${songInfo.ucid}`
+    };
+};
+
+const convertSecondsToTime = seconds => new Date(seconds * 1000).toISOString().substr(11, 8);
+
+const getFormatContainersByFormats = formats => {
```

Comment:
how about 
```js
const getFormatContainersByFormats = formats => 
    formats.map(format => format.container).filter(Boolean)
```
It will do the same thing since:
1.  `formats.indexOf(format) === index` is always `true` 
1. `format !== undefined` is the same as `Boolean(format)`
1. `(format) => Boolean(format)` is the same as `Boolean`

Reply:
I dont remember so well but I think there was a reason I did this line:
```js
formats.indexOf(format) === index
```
I think its not always true but I will check it.

Reply:
forget what I said above. your'e a true **pro**